packages/device_info_plus/device_info_plus/macos/Classes/DeviceInfoPlusMacosPlugin.swift

```diff
@@ -18,12 +18,17 @@ public class DeviceInfoPlusMacosPlugin: NSObject, FlutterPlugin {
     }
 
     private func handleDeviceInfo(result: @escaping FlutterResult)-> Void{
+        let osVersion = ProcessInfo.processInfo.operatingSystemVersion;
+
```

Comment:
Is the whitespace needed?

```suggestion
        let osVersion = ProcessInfo.processInfo.operatingSystemVersion;
```

Reply:
Removed, and also moved the osVersion closer to where it's used